packages/teams-js/src/public/authentication.ts

```diff
@@ -358,9 +348,11 @@ export namespace authentication {
     // Ensure that the new window is always smaller than our app's window so that it never fully covers up our app
     width = Math.min(width, Communication.currentWindow.outerWidth - 400);
     height = Math.min(height, Communication.currentWindow.outerHeight - 200);
+
     // Convert any relative URLs into absolute URLs before sending them over to the parent window
-    const link = document.createElement('a');
-    link.href = authenticateParameters.url.replace('{oauthRedirectMethod}', 'web');
+    const qualifiedURL = fullyQualifyUrlString(authenticateParameters.url.replace('{oauthRedirectMethod}', 'web'));
```

Comment:
>qualifiedURL [](http://example.com/codeflow?start=10&length=12)

I don't actually care about this but it's kind of funny that you called this `fullyQualifiedURL` above but just `qualifiedUrl` here.

Reply:
Sometimes I like to leave little traps in the PR for you to find